tests/resources/applications/maven/liberty-maven-test-app/src/main/liberty/config/server.xml

```diff
@@ -13,7 +13,7 @@
 -->
 <server description="Sample Servlet server">
     <featureManager>
-        <feature>jsp-2.3</feature>
+        <feature>jakartaee-10.0</feature>
```

Comment:
If we're actually starting the server (as I think we do in some of these tests), using the full feature will add significantly more time.   Better to stick with the EE 9/10-equivalent:  pages-3.0/3.1